src/main/java/org/gridsuite/ds/server/model/ResultEntity.java

```diff
@@ -46,4 +47,7 @@ public ResultEntity(UUID id, UUID timeSeriesId, UUID timeLineId, DynamicSimulati
     @Enumerated(EnumType.STRING)
     private DynamicSimulationStatus status;
 
+    @Column(name = "outputState")
+    private byte[] outputState;
```

Comment:
It's a problem to have the file content in the same entity and same table. See dynamicSimulationService.findStatus(). You request the whole file just to get the computation status

Reply:
This problem can not avoid since using bytea, but we have only some MB, in comparison to a network, it is not a real problem

Reply:
A solution is using projection with jpql : https://github.com/gridsuite/dynamic-mapping-server/blob/main/src/main/java/org/gridsuite/mapping/server/repository/RuleRepository.java#L22
We can enhance in the next PR..